examples/seq2seq/seq2seq.py

```diff
@@ -161,8 +161,12 @@ def main():
                         help='GPU ID (negative value indicates CPU)')
     parser.add_argument('--resume', '-r', default='',
                         help='Resume the training from snapshot')
-    parser.add_argument('--unit', '-u', type=int, default=1024,
+    parser.add_argument('--unit', '-u', type=int, default=512,
```

Comment:
ah, I see.  I will fix it.
However, I think it is not necessary to follow because the tutorial does not show any final result (BLEU score).